tests/test_tables.py

```diff
@@ -267,8 +284,13 @@ def test_where_predicates(t):
     """)
 
 
-def test_sort(t):
+def test_sort(table):
+    t = table
+    t['totals'] = t['points'] * t['count']
+    print("test_sort")
```

Comment:
We should probably take away the print statements for the final push to master to ensure the code is clean.